File: c5/simpleMath.c
#include <stdio.h>
#include <math.h>

int main()
{
	int a;
	int b;

	printf("What is the first number? ");
	scanf("%d", &a);
	printf("What is the second number? ");
	scanf("%d", &b);
	printf("%d + %d = %d\n",a, b, a + b);
	printf("%d - %d = %d\n",a, b, a - b);
	printf("%d * %d = %d\n",a, b, a * b);
	printf("%d / %d = %d\n",a, b, a / b);

	return 0;
}

File: py5/simpleMath.py
#!/usr/bin/env python2.7
import sys


def uInputInt(prompt):
    sys.stdout.write(prompt)
    while True:
        try:
            out = int(sys.stdin.readline())
        except:
            pass

        sys.stdout.write("That was not an int!\n")

    return out

def main():
    fnum = uInputInt("What is the first number? ")
    snum = uInputInt("What is the second number? ")
if __name__ == "__main__":
    main()
